bedrock/thunderbird/templates/thunderbird/start/release.html

```diff
@@ -34,6 +72,7 @@ <h2 class="iconic">{{ _('Need Support?') }}</h2>
           <ul>
             <li><a href="{{ url('thunderbird.features') }}">{{ _('Features') }}</a></li>
             <li><a href="https://blog.mozilla.org/thunderbird/">{{ _('Visit our blog') }}</a></li>
+            <li><a href="https://donate.mozilla.org/thunderbird/about/">{{ _('Donate') }}</a></li>
```

Comment:
Well, as it was in the menu I thought maybe it should be "with context". Anyway, I changed it to just /thunderbird/